### ISSUE TITLE
[SUG] Artifact retrieval becomes a roundend condition for UNSC

### ISSUE THREAD

**Issue by ghost** [HaloSpaceStation/HaloSpaceStation13#2688]
The round should probably end if the UNSC manage to bring the artifact into their base, citing a recent round where they managed this but entered a deadlock state where neither side could really do much for a while until the end-fleet killed the covenant flagship, it sort of unnecessarily dragged out the round.

**Comment by 24THBN** [comment]
You can assault the ONI base.

**Comment by Jaybirdyy** [comment]
I'd argue and say they just shouldn't be allowed to bring it into their base, or give us an easier way to get in.  It is really cheesy, and indeed does lock the round to nothing.  It is pretty much the covenants only objective, and they can just take it into a base that requires set methods to even get into.

**Comment by ghost** [comment]
Round ends once the covenant get the artifact back to the ship, as then they can then end the round at their leisure by glassing geminus, which is a process that I have never seen stopped by any last marines, as by this point the entire UNSC has been wiped out, vehicles and all. When this situation happens in the reverse, when the covenant are decimated and the UNSC have completely taken the win, they have to sit and wait for the fleet to arrive to end the round for them, rather than being able to expedite the process. I don't see why the same shouldn't apply for the UNSC endgame in this case - at the moment it feels like time padding as when either side has extracted the artifact, their enemy can be considered defeated. The artifact hunt is more or less the last competition present in the round and both sides expend themselves completely while trying to grab it

tl;dr covenant have an endround button after getting artifact, I think UNSC should too.

**Comment by 24THBN** [comment]
But UNSC already lacks the means to locate the relic on their own.
Most rounds can end with forces still being available, most if not all UNSC victories end because of the round end vote unless they pull a maneuver on the Covenant flagship, which they can do by flying outside of it and breaking in, the fact that they decide not to shouldn't limit their option for do so, just as how Covenant could make a comeback assaulting ONI.

**Comment by ghost** [comment]
the UNSC are completely capable of taking artifact scanners from the covenant and getting the artifact themselves if they overrun a scanning point, they can therefore acquire the means to locate the relic and nab it. To clarify, I'm not saying it should be an instant roundend, but a process that still takes a little time can can potentially be interrupted like glassing, though obviously difficult to interrupt. 

Easiest way might just be a timer starting when the artifact enters the base. Covenant have to assault within that timer or the round ends, likely with a UNSC victory.

**Comment by Jaybirdyy** [comment]
Yeah, I changed my mind.  It would probably be nice.  I just hope if that does become a round end possibility, that we can access the base a bit easier.

**Comment by caelaislinn** [comment]
#2713 prevented artifacts being taken to ONI base, this will depend on the implementation of phase 3
